frontend/src/pages/SettingsPage.tsx

```diff
@@ -65,55 +124,89 @@ export default () => {
     <div className="h-96">
       <NavBar />
       <div className="flex justify-center inset-x-0 flex-col grow h-full space-y-8 w-1/3 m-auto mt-5">
-        <Card variant="outlined" className="h-full">
+        <Paper variant="outlined" className="h-full flex-grow-1 flex-col grow">
           <Box
             className="h-full w-full"
+            flexGrow="1"
             sx={{ borderBottom: 1, borderColor: "divider" }}
           >
-            <Tabs value={value} onChange={handleChange}>
+            <Tabs value={value} onChange={handleChangeTab}>
               <Tab label="Profile Settings" {...a11yProps(0)} />
-              <Tab label="Account Settings" {...a11yProps(1)} />
+              <Tab label="Delete Account" {...a11yProps(1)} />
+              <Tab label="Change Password" {...a11yProps(2)} />
             </Tabs>
             <TabPanel value={value} index={0}>
               Profile stuff
             </TabPanel>
             <TabPanel value={value} index={1}>
-              <Button
-                className="mb-5"
-                onClick={() => {
-                  setIsDialogOpen(true);
-                }}
-              >
-                Delete Account
-              </Button>
+              {deleteAlertStatus === 0 ? (
+                  <Alert severity="warning" sx={{ marginBottom: "1rem" }}>
+                    Warning, you are about to delete your account
+                  </Alert>
+                ) : deleteAlertStatus === 1 ? (
+                  <Alert severity="error" sx={{ marginBottom: "1rem" }}>{deleteAlertMessage}</Alert>
+                ) : (
+                  <Alert severity="success" sx={{ marginBottom: "1rem" }}>
+                    {deleteAlertMessage}
+                  </Alert>
+              )}
+              <Typography sx={{  marginBottom: "6rem" }}>To proceed with the account deletion, please enter password here:</Typography>
+              <Box flexDirection="row">
```

Comment:
Is it not possible to use tailwind CSS?
E.g. className="mb=-4"

mb-4 | margin-bottom: 1rem; /* 16px */

Reply:
https://tailwindcss.com/docs/margin